TACC/Core-CMS
# refactor: less calls to `add_perm`

## Overview

Use less cals to `add_perm()` by consolidating them into `utils.py`.

## Related

- [CMD-189](https://tacc-main.atlassian.net/browse/CMD-189)

## Changes

- **changed** all group_perms to use util functions

## Testing

1. Create/Open a Core-CMS instance.
2. In the docker, run:
    - `python manage.py set_group_perms grid_editor_basic grid_editor_advanced text_editor_basic text_editor_advanced media_editor_basic media_editor_advanced news_writer_basic news_writer_advanced`
3. In CMS admin UI, verify those 8 groups are created.
4. Verify the groups each have relevant permissions.
    <sup>I merely skimmed the results.</sup>

## UI

…
